src/sentry/static/sentry/app/views/organizationIntegrations/integrationIcon.tsx

```diff
@@ -21,7 +21,16 @@ const Icon = styled('img')<IconProps>`
 
 const IntegrationIcon = ({integration, size}: Props) =>
   integration.icon ? (
-    <Icon size={size} src={integration.icon} />
+    <Icon
+      size={size}
+      src={integration.icon}
+      onError={e => {
+        (e.target as HTMLImageElement).src =
+          (integration.provider.key !== undefined &&
```

Comment:
@NisanthanNanthakumar When would `integration.provider.key` be undefined? I think in our TS definitions this field is always a string.

Reply:
@scefali Sorry, not too familiar with TS, but what if the object does not have the `key` property. There is no default value right?

Reply:
@NisanthanNanthakumar If the object doesn't have the `key` property, a bunch of other things would break. The `key` will always be there. You can look at the `IntegrationSerializer` to see for yourself.